src/components/add_book.js

```diff
@@ -10,10 +13,23 @@ const AddBook = () => {
       [e.target.name]: e.target.value,
     });
   };
+
+  const dispatch = useDispatch();
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (title.trim().length === 0 || title.trim().length === 0) {
+      alert('Please fill missing fields');
+      return;
+    }
```

Comment:
**[REQUIRED CHANGES]**

- Hey, Please kindly do have in mind that using the `alert()` method is regarded as [not a good practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). You could either prevent the form from submitting an empty field or manually create and style some error message. Please, kindly re-evaluate this part of the code.
- Also, I think you wanted to check for the title and author field here:

` if (title.trim().length === 0 || title.trim().length === 0) {
      alert('Please fill missing fields');
      return;
    }`